src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
@raquelromano-gov Actually, the other reason I did this is because there are other places where the login status is set to true or false and I didn't want to put an extra dispatch in each place along with it.